### DISCUSSION TITLE
Starting sfcap

### DISCUSSION THREAD

**Discussion by RobertoGreiner** [phaag/nfdump#463] [General]
Hi,

I'm setting up nfdump to collect flow, but my switch only generates sFlow data. I know that nfdump can collect sFlow using sfcapd, but I couldn't find anywhere the information on how to tell nfdump that it should start sfcapd instead of nfcapd. Could someone help me on that?

I'm using Ubuntu 22.04, with nfdump and sfcap installed using apt. (nfdump and nfdump-sflow respectively). 

Thank you,

Roberto

**Comment by phaag** [comment]
nfdump does *not* start any collector, neither nfcapd nor sfcapd. You have to start them individually at the command list or with a start script. The option of sfcapd are compatible with nfcapd.
